src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -16,16 +17,12 @@ export const WebhookReceiverPost = async (request: Request, response: Response):
 	const id = request.headers["x-github-delivery"] as string;
 	const uuid = request.params.uuid;
 	const payload = request.body;
-	let webhookSecret: string;
 	try {
-		const gitHubServerApp = await GitHubServerApp.findForUuid(uuid);
-		if (!gitHubServerApp) {
+		const webhookSecret = await getWebhookSecret(uuid);
+		if (!webhookSecret) {
```

Comment:
instead of checking if webhookSecret is defined, the function should throw if it doesn't, which will be caught in this try catch. However, I don't think we should return a 500.  Instead, it should return a 400 or a 404